src/main/templates/net/imagej/ops/Ops.list

```diff
@@ -104,6 +104,7 @@ namespaces = ```
 		[name: "paddingIntervalOrigin",          iface: "PaddingIntervalOrigin"],
 		[name: "sigma",                          iface: "Sigma",               aliases: ["sigmaFilter", "filterSigma"]],
 		[name: "variance",                       iface: "Variance",            aliases: ["varianceFilter", "filterVariance", "var", "varFilter", "filterVar"]],
+		[name: "frangiVesselness",			 iface: "FrangiVesselness"],
```

Comment:
This should now go to `f` in the alphabet (and indentation fixed), right?

Reply:
Oops. You are right. I will change it here and in the namespace.